core/vm/runtime/runtime.go

```diff
@@ -113,6 +113,14 @@ func Execute(code, input []byte, cfg *Config) ([]byte, *state.StateDB, error) {
 		vmenv   = NewEnv(cfg)
 		sender  = vm.AccountRef(cfg.Origin)
 	)
+	if cfg.ChainConfig.IsYoloV2(vmenv.BlockNumber) {
+		cfg.State.AddAddressToAccessList(cfg.Origin)
+		cfg.State.AddAddressToAccessList(address)
+		for _, addr := range vmenv.ActivePrecompiles() {
+			cfg.State.AddAddressToAccessList(addr)
```

Comment:
@holiman is there a specific reason that this line is called twice?

Reply:
Whoa, nope. It doesn't cause any bad side-effects, but that's definitely a blooper

Reply:
:) of course I didn't mentioned this as critical, but mostly as a blooper.

Do you want me to make a PR or you are going to handle it in one of your next PRs?

Reply:
You'd be very welcome to, if you feel like it! Otherwise I'll fix it sooner or later

Reply:
Here it is @holiman. #21956 
Thanks